tests/nanocoap_cli/Makefile

```diff
@@ -26,6 +26,7 @@ LOW_MEMORY_BOARDS := \
   nucleo-f302r8 \
   saml10-xpro \
   saml11-xpro \
+  sipeed-longan-nano \
```

Comment:
Unfortunatly, this board has not enough memory :worried:

Reply:
Yea you can only have either `nanocoap_vfs` or ` nanocoap_dtls` on this one.